web/ui/dashboard/src/app/private/pages/project/event-logs/event-logs.component.ts

```diff
@@ -92,7 +92,7 @@ export class EventLogsComponent implements OnInit {
 
 	async ngOnInit() {
 		const data = this.getFiltersFromURL();
-		this.checkIfTailModeIsEnabled(data);
+		this.checkIfTailModeIsEnabled() ? this.refetchEvents(data) : this.getEventLogs({ ...data, showLoader: true });
```

Comment:
Do you really need to prefetch events?
You can just start the retry interval instead.

Reply:
I added this for the loader effect, it's really just for user experience

Reply:
You want to show the loader effect after enabling the tail mode?